src/quic/node_quic_session.cc

```diff
@@ -1976,31 +1998,12 @@ bool QuicSession::ReceiveStreamData(
     const uint8_t* data,
     size_t datalen,
     uint64_t offset) {
-  auto leave = OnScopeLeave([&]() {
-    // This extends the flow control window for the entire session
-    // but not for the individual Stream. Stream flow control is
-    // only expanded as data is read on the JavaScript side.
-    // TODO(jasnell): The strategy for extending the flow control
-    // window is going to be revisited soon. We don't need to
-    // extend on every chunk of data.
+  auto leave = OnScopeLeave([=]() {
```

Comment:
Any reason for the switch to `=`?

Reply:
Oh, errant edit, I was just testing something. will revert that